### PR TITLE
chore/9995-Usei18nInUnitTests-Home-3

### PR DESCRIPTION
**Pull request by ATeal**
## Description of Change
Update the unit test in [this issue](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/9995) to use i18n.

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx [hunk 1]
```diff
@@ -14,29 +16,16 @@ context('IncorrectServiceInfo', () => {
...
+    expect(screen.getByText(t('militaryInformation.incorrectServiceInfo.body.2'))).toBeTruthy()
+    expect(screen.getByText(t('militaryInformation.incorrectServiceInfo.body.3'))).toBeTruthy()
+    expect(screen.getByRole('link', { name: displayedTextPhoneNumber(t('8005389552')) })).toBeTruthy()
   })
 
   it('should call DMDC on press', () => {
-    fireEvent.press(screen.getByRole('link', { name: '800-538-9552' }))
-    expect(Linking.openURL).toHaveBeenCalledWith('tel:8005389552')
+    const i18nPhoneNumber = t('8005389552')
+    fireEvent.press(screen.getByRole('link', { name: displayedTextPhoneNumber(i18nPhoneNumber) }))
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx:28]: No need for the const, you can just pass t('800... here.

Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx:28]:
```suggestion
    fireEvent.press(screen.getByRole('link', { name: displayedTextPhoneNumber(t('8005389552') }))
```

File: VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx [hunk 2]
```diff
@@ -14,29 +16,16 @@ context('IncorrectServiceInfo', () => {
...
+    expect(screen.getByText(t('militaryInformation.incorrectServiceInfo.body.1'))).toBeTruthy()
+    expect(screen.getByText(t('militaryInformation.incorrectServiceInfo.body.2'))).toBeTruthy()
+    expect(screen.getByText(t('militaryInformation.incorrectServiceInfo.body.3'))).toBeTruthy()
+    expect(screen.getByRole('link', { name: displayedTextPhoneNumber(t('8005389552')) })).toBeTruthy()
   })
 
   it('should call DMDC on press', () => {
-    fireEvent.press(screen.getByRole('link', { name: '800-538-9552' }))
-    expect(Linking.openURL).toHaveBeenCalledWith('tel:8005389552')
+    const i18nPhoneNumber = t('8005389552')
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx:27]:
```suggestion
```

File: VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx [hunk 3]
```diff
@@ -14,29 +16,16 @@ context('IncorrectServiceInfo', () => {
...
+    fireEvent.press(screen.getByRole('link', { name: displayedTextPhoneNumber(i18nPhoneNumber) }))
+    expect(Linking.openURL).toHaveBeenCalledWith(`tel:${i18nPhoneNumber}`)
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.test.tsx:29]:
```suggestion
    expect(Linking.openURL).toHaveBeenCalledWith(`tel:${t('8005389552')}`)
```

**Review by dumathane — CHANGES_REQUESTED** [review]
Please drop the const declaration, otherwise this looks good.

**Comment by ATeal** [comment]
> Please drop the const declaration, otherwise this looks good.

I went ahead and made the change. I originally had it that way but abstracted to avoid calling the `t(` function multiple times for the same string value. Nevertheless it's in :)

**Review by rbontrager — APPROVED** [review]

**Pull request merged**